src/containers/Symptoms/Symptoms.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+// * Page
+import SymptomsList from "../../components/Pages/Symptoms/SymptomsList";
+
+function Symptomps() {
+  return (
+    <div>
+      <SymptomsList />
+    </div>
+  );
+}
+
+export default Symptomps;
```

Comment:
In the container for Symptoms, we just need to import and use the List of Symptoms 👻

Reply:
Great!